src/providers/CeloMappingProvider.ts

```diff
@@ -0,0 +1,11 @@
+import { MappingProvider } from './MappingProvider'
+import { MappedTokenData } from '../constants/types'
+import celoMappings from '../local_mappings/celo.json'
+
+// barring a better external source, we use a local source of mappings for avalanche
```

Comment:
comment seems unrelated
```suggestion
// barring a better external source, we use a local source of mappings for celo
```